docs/cli/descheduler.md

```diff
@@ -22,6 +22,12 @@ descheduler [flags]
       --disable-metrics                          Disables metrics. The metrics are by default served through https://localhost:10258/metrics. Secure address, resp. port can be changed through --bind-address, resp. --secure-port flags.
       --dry-run                                  Execute descheduler in dry run mode.
       --enable-http2                             If http/2 should be enabled for the metrics and health check
+      --feature-gates mapStringBool              A set of key=value pairs that describe feature gates for alpha/experimental features. Options are:
```

Comment:
⚠️ I'm leaning towards not registering the log feature flags. let me know what you think 🤔 

`ContextualLogging` sounds interesting but the flag is a bit misleading here. It makes it sound like it's a Desheduler feature flag, but it's rather the logs api feature flags.

Reply:
Met with @knelasevero and we decided to pursue logsapi feature flags in a separate Issue so we can be intentional about adding new functionality.